src/components/UI/layouts/Container/Container.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
After re-reading atomic structure I think this will be an atom. But for brevity we can put the layout folder in the atoms so it will be `atoms/layout/Container/ Container`